homeworks/rostyslav.havuka_RostyslavHavuka/13-ng-forms/forms/src/app/password-input-component/password-input-component.component.html

```diff
@@ -0,0 +1,4 @@
+<label for="password">{{ labelPassword() }}</label>
+<br>
+<input placeholder="Enter password" name="password" [(ngModel)]="value"  (ngModelChange)="updateValue($event)" [type]="type"/>
```

Comment:
Reformat the code. You have many attributes here so please move each one to the next line to save readability